python/packages/autogen-core/src/autogen_core/application/_helpers.py

```diff
@@ -72,3 +72,7 @@ def _build_for_new_topic(self, topic: TopicId) -> None:
         for subscription in self._subscriptions:
             if subscription.is_match(topic):
                 self._subscribed_recipients[topic].append(subscription.map_to_agent(topic))
+
+    @property
+    def subscriptions(self) -> List[Subscription]:
```

Comment:
I am not sure if we are ready to expose the subscriptions, subscription managers, and servicers as public API yet... There can be different implementations of subscription manager for example, e.g., based on Redis, and we haven't designed their public APIs. So, let's leave these private for now.